jobs/pull_requests.rb

```diff
@@ -31,14 +31,14 @@
 	trend = GithubDashing::Helper.trend_percentage_by_month(prev, current)
 	trend_class = GithubDashing::Helper.trend_class(trend)
 
-	send_event(
-		'pull_requests', 
-		{
-			series: series, # Prepare for showing open/closed stacked
-			displayedValue: current,
-			difference: trend,
-			trend_class: trend_class,
-			arrow: 'icon-arrow-' + trend_class
-		}
-	)
-end
\ No newline at end of file
+  send_event(
+    'pull_requests',
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.<br>Tab detected.